randovania/games/am2r/logic_database/The Tower.txt

```diff
@@ -506,7 +511,7 @@ Extra - liquid_info: {'liquid_type': 0, 'liquid_level': 0}
   > Middle
       Any of the following:
           Space Jump
-          Hi-Jump Boots and Morph Glide (Advanced)
+          Hi-Jump Boots and Morph Ball and Morph Glide (Advanced)
```

Comment:
what about spider along the ceiling, fall down, then morph glide?

Reply:
![image](https://github.com/randovania/randovania/assets/125271738/7faa1b8a-df7d-4c8e-9b18-e7007b2552ca)
as this only would ever get you halfway, I don't see the worth in adding it

Reply:
~~but what if you start in the middle~~
Fair enough.